draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -388,7 +388,8 @@ congestion control algorithms.
 In contexts where differing congestion control
 algorithms are used, it is important to understand whether
 the proposed congestion control algorithm could result in more
-harm than previously defined algorithms to flows sharing a common bottleneck.
+harm than previous standards-track algorithms (e.g. {{!RFC5681}},
+{{!RFC9002}}, {{!RFC9438}}) to flows sharing a common bottleneck.
 The measure of harm is not restricted to the equality
```

Comment:
You present this list an example, as in "example given, (classic), (Reno in QUIC), Cubic". I actually think that this list is exhaustive.

Reply:
It is exhaustive, but maybe won't be in the future? I could go either way on this.

Reply:
I think the current list with e.g. list is fine - the list isis for previous standards-track RFCs fro algo's, so if this were extended in the future, it would be OK for the advice to include those to-be-done RFCs.

Reply:
Yeah, I was trying to be consistent with the approach in the next paragraph, which has a similar list: "A proposed congestion control algorithm SHOULD be evaluated when competing using standard IETF congestion controls, e.g. {{!RFC5681}}, {{!RFC9002}}, {{!RFC9438}}."

Reply:
OK, I think I was just too picky.